vrs/vrs_annotation_batch.py

```diff
@@ -297,6 +300,11 @@ def main(args):
             f"gs://gnomad-vrs-io-finals/ht-outputs/annotated-checkpoint-VRS-{prefix}.ht"
         )
 
+        # Annotate final Hail Table with GA4GH version
+        ht_annotated = ht_annotated.annotate_globals(
+            ga4gh_vrs_version=ga4gh.vrs.__version__
```

Comment:
in the quick fix and in the google doc of descriptions, this was just "vrs_version"

Reply:
corrected in commit coming in a few minutes